common/logisticspipes/modules/ModuleActiveSupplier.java

```diff
@@ -190,21 +191,26 @@ public ItemIdentifierInventory getDummyInventory() {
 	}
 
 	@Override
-	public void tick() {
-		if (!_service.isNthTick(100)) {
+	public void tick()
+	{
```

Comment:
Put braces on the same line (or in general, as @theZorro266 already said, keep the formatting consistent with the rest of the code)